Sources/StreamChatSwiftUI/ColorPalette.swift

```diff
@@ -50,6 +50,7 @@ public struct ColorPalette {
     public var highlightedAccentBackground: UIColor = .streamAccentBlue
     public var highlightedAccentBackground1: UIColor = .streamBlueAlice
     public var pinnedBackground: UIColor = .streamHighlight
+    public var staticWhiteBackground: UIColor = .streamWhiteStatic
```

Comment:
We should not have colour names in these properties. Since the goal of these are meant to be replaced by other colors. We should instead add context on where it is used, like `voiceRecordingShadowBackground`